compose.yml

```diff
@@ -37,6 +37,7 @@ services:
       - mnestix-database:/app/prisma/database
 
   mnestix-api:
+    # TODO update to 1.2.0 as soon as it's released
     image: mnestix/mnestix-api:1.1.0
```

Comment:
If anybody is using an older mnestix-browser version, they would then always use the latest mnestix-api.
This would also occour on breaking changes, so an explicit versioning is more stable (and less work, as we dont have to test every browser <-> api setup)